draft-ietf-cdni-uri-signing.xml

```diff
@@ -1262,6 +1263,37 @@ http://*/folder/content-83112371/quality_*/segment????.mp4
         number for this document.]</t>
 
       </section>
+
+      <section anchor="sec.IANA.cdnistt"
+               title="CDNI URI Signing Signed Token Transport">
+        <t>The IANA is requested to create a new "CDNI URI Signing
+        Signed Token Transport" subregistry in the "Content
+        Delivery Networks Interconnection (CDNI) Parameters" registry.
+        The "CDNI URI Signing Signed Token Transport"
+        namespace defines the valid values
+        that may be included in a URI Signing token.  Additions to the
```

Comment:
I am not sure that "URI Signing token" is the defined term in the specification. Maybe use "signed JWTs" or "Signed Token"?